### ISSUE TITLE
"I'm sorry, I didn't understand my server's response"

### ISSUE THREAD

**Issue by x1sspic** [ivee/ivee#5]
As the subject says, " I'm sorry, I didn't understand my server's response"  is all that Ivee says anymore.  I tried the beta because she seemed to have trouble understanding me.  She is now an expensive clock and can't do anything.  Even a reset doesn't do anything.

**Comment by johnnygoodman** [comment]
I'm having the same problem. I've power cycled the unit with no effect.

**Comment by AaronOosterbaan** [comment]
I am also getting this message quite frequently. About 50% of the time.

**Comment by iveelover** [comment]
x1sspic====> This is for Beta testers, maybe you don't understand how that works. The point of the upgrade and only for the beta testers is to find the bugs and report them to get fixed. Not condemn the clock.

**Comment by johnnygoodman** [comment]
@iveelover In beta tests I do for other companies and my own we receive a product or feature, use it and report defects. The reported defect here is that other than the clock function, no other command is accepted. Isn't this legit and of concern? 

If not, please do explain what the scope of testing involves or link to where this is described.

**Comment by iveelover** [comment]
You are correct and posted correctly but in my opinion "x1sspic" posted as a user not happy with a product. It doesn't help if you just don't report the facts and the problem, not tell people not to do something that we are hear to do. If that makes sense. This is a very extensive program with many outlets and the facts only need to be reported. Or the project won't move forward in fixing things. That's why we have this forum and not posting in the other public forum.

Oh and note to, x1sspic, tried to use the beta program to fix a problem that was fixed two weeks ago. "I tried the beta because she seemed to have trouble understanding me" x1sspic should have followed the fix on that before upgrading to the beta now you have two problems going on for one test.

**Comment by johnnygoodman** [comment]
Ok, given that, questions:
1. Perhaps there's confusion on which version is being run. How can I confirm which version I am on given that the ivee is not responding to the commands "What's your current system version?" or "System update"?
2. If I am indeed on the correct beta version, which commands should respond successfully and which should result in "I'm sorry, I didn't understand my server's response" while beta?
3. Is there a method to revert the system from beta to non-beta should a beta tester wish to do so? 
4. Is this the format you want feedback in? If not, please provide an example.

**Comment by x1sspic** [comment]
@iveelover, I think you are misunderstanding.  I like the product quite well.  I signed up for beta testing because of inconsistent issues with it understanding voice commands (mostly an annoyance), but that doesn't mean I was condemning the product by any means.  

The bug I'm reporting with the beta is an error that has essentially rendered Ivee nonfunctional, which is quite frustrating!   Reporting this bug and my frustration with it is not the same as condemning the product.  I understand how beta testing works.

**Comment by iveelover** [comment]
The beta from what I understand is "version 43" for the test and is the first round of the test. As for finding out what version just ask her "Hello Ivee, followed by "What Version" she should tell you what she is running. I just did it with mine to be sure. I think it's important that two weeks ago she had the problem of saying she couldn't understand he server. Now in the test she is saying that she can't hear you. Big difference in the response. I do get her to answer questions but she won't control anything other then saying "i can't hear you" "are you there" so the bug is something is blocking the comand going through. But she is indeed talking to the server. She is pulling the weather and when I ask "tell me something funny" she tells me and she could only get that info from the server. I did have to go in and redo the "my location" because she lost that info when I updated to version 43. But now she gives the info on my true location. 

On a roll back you might factory reset but your Mac is recorded with the company on getting beta versions so you would have to have them take you off the list.

I think you might think I work for the company I don't I am a tester too. But admittedly a big fan of Ivee!

**Comment by ITATodd** [comment]
I think that is what he is saying, it won't let him ask what version because every command is coming back with "I'm Sorry, I don't understand my servers response." So, since I don't see a way ONLINE via the dashboard to see this, which I personally think we should be able to control features of IVEE via the dashboard as well. 

So, he's left wondering HOW he can find out what version IVEE is using. So, hopefully someone will respond with a fix or push it out to his device. I'm sure that have all of our MAC addresses and should be able to push up new firmware as needed. 

As for being a beta tester, that's what it's all about, like right now I can't control any of my IRIS devices, there's another BUG to fix. We report them and then fix them. Be nice to have some interaction from an IVEE rep like we to have in other BETA Forums.

**Comment by bensilver** [comment]
Hi all,we are working on your query and should have a response shortly. We have received all of your comments, so are working as fast as possible to fix them.

**Comment by lancemoreland** [comment]
Same issue for me.  I just updated Ivee to the beta version and the response to everything is "I'm sorry, I didn't understand my server's response" This seems to happen after every update.

**Comment by dusterp** [comment]
I tried disconnecting from the internet by holding the command bar down for 5sec and then saying factory reset.  After I got it set back up it's still not working.

**Comment by dusterp** [comment]
@bensilver any updates?  It's been 7 days since your last comment.  I'm still stuck in alarm/clock only mode.

**Comment by lancemoreland** [comment]
Are we going to get an update?

**Comment by x1sspic** [comment]
I'm starting to fear we're SOL. Support issued downgrade for my unit. Problem is, Ivee won't do anything anymore, including recognizing the "software update" command! I suspect unless they fix something on the server side for version 43, we're stuck with one heck of an expensive clock.

**Comment by dusterp** [comment]
Post to their Facebook that can some time spark a response.

**Comment by bensilver** [comment]
In order to initiate a system update you need to press the command bar on the ivee. 
If anyone else wishes to roll back please email me.
There are issues on the Wit.ai side that we are working with them on to iron out.

Many thanks
Ben

**Comment by iveelover** [comment]
Hello all,
Here's a new list of problems and unless I am missing something on a update here they are...

1- She won't set alarm two.
2- She refuses to change alarm sounds and resets the alarms when changing the sounds.
3- She is back to controlling the "NEST" ok
4- Both Ivee's I have one in beta and the other seem since the last up date to be hard of hearing!

Not sure what you mean by holding the control bar for an update. Hold or push the bar then what?

A question here! Why is staples dropping the Ivee? and why didn't we get that great deal? I spent full price on mine [X2] and then they went on clearance sale at staples for $29.99. I called to buy four of them and they said they were sold out and not getting more through out the USA! Also is the Flex Ivee off the market now too?

Thanks

**Comment by x1sspic** [comment]
@bensilver : I tried pressing the command bar.  It's the SAME response.  "I'm sorry, I didn't understand my server's response, please try again."  Even after performing a factory reset and starting from scratch, she still has the same response.  

@anyone: has anyone else that updated to the beta version had any success?  Having Ivee rendered completely inoperative is a pretty big bug that I would consider pulling the beta for until the issue is sorted out.

**Comment by dusterp** [comment]
No luck here

**Comment by x1sspic** [comment]
Support was able to successfully downgrade Ivee to version 42 and is working now.  Because Ivee was not recognizing any commands, I had to do a factory reset.  When connecting to the internet after reset Ivee does a check for software update.  

As a side, In order to factory reset, I had to press and hold the command bar until Ivee disconnected from the internet, then the device is able to recognize the "factory reset" command

**Comment by lancemoreland** [comment]
I am back on ver. 42 and it was working........now all I get is the usual   "I'm sorry, I didn't understand my server's response"

**Comment by lancemoreland** [comment]
It looks like this has gone dead.  No updates, no improvements, no comments from Ivee.  I removed Iris and my Nest thermostats from Ivee and it is stable but very limited in what it can do.  I guess Ivee can't handle anything outside of the box.  Very disappointing.  Going on 4 months now with no improvement.
